src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -219,8 +222,7 @@ protected virtual Task RequestExecutedAsync(in GraphQLRequestExecutionResult req
             return Task.CompletedTask;
         }
 
-        private Task WriteErrorResponseAsync(HttpResponse httpResponse, IGraphQLSerializer serializer, CancellationToken cancellationToken,
-            string errorMessage, HttpStatusCode httpStatusCode)
+        protected virtual Task WriteErrorResponseAsync(HttpContext context, string errorMessage, HttpStatusCode httpStatusCode = HttpStatusCode.BadRequest)
```

Comment:
```suggestion
        protected virtual Task WriteErrorResponseAsync(HttpContext context, string errorMessage, HttpStatusCode httpStatusCode)
```

That is why in one of the previous PR, I removed the default value.

Reply:
We have tests but GitHub Actions are broken for now.

Reply:
Let's merge the other PR and then you can work on this one.

Reply:
fixed to not be optional